README.md

```diff
@@ -63,6 +67,16 @@ To run the tests and see the results in your terminal, run:
 bundle exec rake jasmine:ci
 ```
 
+## Issue with FFI on OSX Mohave
```

Comment:
Nitpick, but I think apple anglicise Mohave as "Mojave" in the name of their OS.